components/symbl_ai/actions/post-formatted-transcript/post-formatted-transcript.mjs

```diff
@@ -0,0 +1,68 @@
+import symblAIApp from "../../symbl_ai.app.mjs";
+import constants from "../constants.mjs";
+
+export default {
+  key: "symbl_ai-post-formatted-transcript",
+  name: "Create Formatted Transcript",
+  description: "Create formatted transcript from the Conversation. See the doc [here](https://docs.symbl.ai/docs/conversation-api/transcript/).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    symblAIApp,
+    conversationId: {
+      propDefinition: [
+        symblAIApp,
+        "conversationId",
+      ],
+    },
+    contentType: {
```

Comment:
Maybe we can move this prop to `app` in order to reuse it in future actions?

Reply:
Cool... That's updated in e3e6191f. Thanks for the suggestion.